pkg/router/template/router.go

```diff
@@ -430,22 +474,23 @@ func (r *templateRouter) writeDefaultCert() error {
 func (r *templateRouter) watchMutualTLSCert() error {
 	caPath := os.Getenv("ROUTER_MUTUAL_TLS_AUTH_CA")
 	if len(caPath) != 0 {
-		reloadFn := func() {
-			log.V(0).Info("reloading to get updated client CA", "name", caPath)
+		r.haveClientCA = true
+		caUpdateChannel := make(chan struct{})
+		crlReloadFn := func(haveCRLs bool) {
+			r.haveCRLs = haveCRLs
+			log.V(0).Info("reloading to get updated client CA CRL", "name", crl.CRLFilename)
```

Comment:
```suggestion
			log.V(0).Info("reloading to get updated client CA CRL", "name", crl.CRLFilename, "have CRLs", haveCRLs)
```
Or is that redundant with the logging that `managedURLs` already does when `nextUpdate` is zero?